aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -89,18 +89,24 @@ module aptos_framework::coin {
     }
 
     /// Capability required to mint coins.
-    struct MintCapability<phantom CoinType> has copy, key, store { }
+    struct MintCapability<phantom CoinType> has copy, store { }
 
     /// Capability required to freeze a coin store.
-    struct FreezeCapability<phantom CoinType> has copy, key, store { }
+    struct FreezeCapability<phantom CoinType> has copy, store { }
 
     /// Capability required to burn coins.
-    struct BurnCapability<phantom CoinType> has copy, key, store { }
+    struct BurnCapability<phantom CoinType> has copy, store { }
 
     //
     // Getter functions
     //
 
+    /// A helper function that returns the address of CoinType.
+    fun coin_addr<CoinType>(): address {
```

Comment:
nit: How about making this public and fully spelling out the name to be "coin_address"?

Reply:
Sounds good. Done.